backend/src/lib/server-side-events.ts

```diff
@@ -274,25 +286,78 @@ export class ServerSideEvents {
       logger.info({ msg: 'event stream close' })
     })
 
-    let lastEventID = 0
-    if (req.headers['last-event-id']) {
-      const last = Number(req.headers['last-event-id'])
-      if (Number.isInteger(last)) {
-        const cookies = parseCookies(req)
-        if (cookies['watch'] === instanceID) {
-          lastEventID = last
-        }
+    // SORT EVENTS INTO SMALLER PACKETS
+    // SO THAT BROWSER PAGE LOADS QUICKER
+    // split events into sections
+    const parts = Object.values(this.events)
+    const start = parts.splice(0, 2)
+    const end = parts.pop()
+    const clusters: ServerSideEvent<unknown>[] = []
+    const policies: ServerSideEvent<unknown>[] = []
+    const agents: ServerSideEvent<unknown>[] = []
+    const infos: ServerSideEvent<unknown>[] = []
+    const addons: ServerSideEvent<unknown>[] = []
+    const remainder: ServerSideEvent<unknown>[] = []
+    parts.forEach((event) => {
+      const data = event.data as WatchEvent
+      switch (data.object.kind) {
+        case 'ManagedCluster':
+        case 'HostedCluster':
+        case 'ClusterDeployment':
+          clusters.push(event)
+          break
+        case 'Policy':
+        case 'PolicySet':
+          policies.push(event)
+          break
+        case 'AgentClusterInstall':
+          agents.push(event)
+          break
+        case 'ManagedClusterInfo':
+          infos.push(event)
+          break
+        case 'ManagedClusterAddOn':
+          addons.push(event)
+          break
+        case 'Application':
+          break
```

Comment:
I think we should probably do this filtering elsewhere, so that we watch the resources so they can be used in the paginated API, but we do not create events for them. And it needs to be coordinated with removing the atoms from the frontend.